maykin_2fa/templates/maykin_2fa/recovery_token.html

```diff
@@ -10,6 +10,19 @@
             recovery codes to verify your identity.
         {% endblocktrans %}</p>
 
+        {% if form.errors %}
```

Comment:
If possible, I would like to see if we can fix this in `maykin_2fa/templates/maykin_2fa/includes/wizard_form.html` (which is included on line 26 in this file). I see that `field.errors` are supposed to be displayed there as well, but those don't seem to work. Could you look into that to see if it can be fixed there? https://github.com/maykinmedia/maykin-2fa/blob/main/maykin_2fa/templates/maykin_2fa/includes/wizard_form.html#L5